firestore/query.go

```diff
@@ -475,6 +644,14 @@ func (q Query) fromProto(pbQuery *pb.RunQueryRequest) (Query, error) {
 		q.limit = limit
 	}
 
+	var err error
+	q.runQuerySettings, err = newRunQuerySettings(nil)
```

Comment:
Is there a case where this line will produce an error? Is there something in the proto that you wanted to extract here?

Reply:
It won't produce error now but I don't want to ignore the error since it may return some error in the future.
Things are getting extracted from proto in the next few lines.